packages/bridge-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [39.1.0]
+
+### Uncategorized
+
+- Revert releases 502.0.0 and 503.0.0 ([#6320](https://github.com/MetaMask/core/pull/6320))
+- Release/503.0.0 ([#6315](https://github.com/MetaMask/core/pull/6315))
+
```

Comment:
These are just development-only changes, so we can omit them from the changelog.

``` suggestion
```